### DISCUSSION TITLE
[help] How does targets handle hash clashes for dynamic targets?

### DISCUSSION THREAD

**Discussion by kendonB** [ropensci/targets#1144] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I notice that the suffixes for dynamic targets appear to be hashes, how does targets handle clashes for these?

**Comment by wlandau** [comment]
Numeric suffixes (https://github.com/ropensci/targets/blob/main/R/class_junction.R#L6), then hashing again (https://github.com/ropensci/targets/blob/main/R/class_pattern.R#L362).
